docs/_data/sql-connectors.yml

```diff
@@ -0,0 +1,86 @@
+avro:
```

Comment:
Missing license header

Reply:
Could you also add a comment explaining how to add a new format or connector. I want it to be really self-evident how to maintain this.